modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -18,6 +18,14 @@ set(TFM_VALID_PARTITIONS
 if (CONFIG_BUILD_WITH_TFM)
   # PSA API awareness for the Non-Secure application
   target_compile_definitions(app PRIVATE "TFM_PSA_API")
+
+  if (CONFIG_TFM_BL1)
+    list(APPEND TFM_CMAKE_ARGS -DBL1:BOOL=ON)
+    if (CONFIG_TFM_BL2_SIGNING_KEY_PATH)
```

Comment:
```suggestion
  if(CONFIG_TFM_BL1)
    list(APPEND TFM_CMAKE_ARGS -DBL1:BOOL=ON)
    if(CONFIG_TFM_BL2_SIGNING_KEY_PATH)
```

Reply:
Old code needs to be fixed but is outside scope of PR, new code must abide by the current zephyr standards (no space between if and bracket)

Reply:
Ok, done.